libcudacxx/include/cuda/std/detail/libcxx/include/cstdlib

```diff
@@ -95,20 +95,18 @@ void *aligned_alloc(size_t alignment, size_t size);                       // C11
 #  include <cstdlib>
 #endif // !_CCCL_COMPILER(NVRTC)
 
+#include <cuda/std/__cstdlib/abs.h>
+#include <cuda/std/__cstdlib/div.h>
+
 _CCCL_PUSH_MACROS
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
 #if !_CCCL_COMPILER(NVRTC)
-using ::div_t;
-using ::ldiv_t;
-using ::size_t;
-#  ifndef _LIBCUDACXX_HAS_NO_LONG_LONG
-using ::lldiv_t;
-#  endif // _LIBCUDACXX_HAS_NO_LONG_LONG
```

Comment:
Q: I assume `_LIBCUDACXX_HAS_NO_LONG_LONG` is just never defined, which is why you did not guard the implementation of `lldiv`?

Reply:
I've discussed this with @miscco and he said the macro is never defined and will be removed in the future.

Reply:
Yeah that macro is only ever defined for `__FREE_BSD__`

Reply:
Great, thx. Here is a tracking issue: #3154